doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -185,17 +202,24 @@ To enable Authenticated Encryption with Associated Data (AEAD), set one or more
 AEAD driver configurations
 ==========================
 
-You can use the following Kconfig options for fine-grained control over which drivers provide AEAD support:
+You can use the following table to check the AEAD algorithm support of each driver:
+
++-----------------------+---------------------------+---------------------------+
+| AEAD cipher           | nrf_cc3xx driver support  | nrf_oberon driver support |
++=======================+===========================+===========================+
+| CCM                   | Supported                 | Supported                 |
++-----------------------+---------------------------+---------------------------+
+| GCM                   | Supported                 | Supported                 |
++-----------------------+---------------------------+---------------------------+
+| ChaCha20-Poly1305     | Supported                 | Supported                 |
++-----------------------+---------------------------+---------------------------+
```

Comment:
I'm a bit confused.

Should we have an row for PSA_WANT_ALG_CCM_STAR_NO_TAG
?

https://github.com/nrfconnect/sdk-zephyr/pull/1227/files#diff-cd9667c11ba0af039229c5929087c5a03509d6b908b66e9eb04d2b7cea77d32dR449

Or maybe a note saying that this variant of the CCM algorithm it is not supported?

Reply:
You are right, I missed to add this. I will add the algorithm.